hazelcast-enterprise-operator/hazelcast-full.yaml

```diff
@@ -1,11 +1,11 @@
-apiVersion: hazelcast.com/v1
-kind: Hazelcast
+apiVersion: hazelcast.com/v1alpha1
```

Comment:
Why did we add alpha back?

Reply:
It was not consistent with the version from Red Hat Container Catalog and Operator Hub. I want to make it consistent. And I think we're more Alpha now. If we need to change it v1, I'd release version `1.0` and change it to `v1`. What do you think?

Reply:
I remember Ted was asking to make v1 otherwise that would be problem during certification process. I am fine with releasing 1.0

Reply:
After the discussion with @leszko , we decided to keep it in alpha. Mostly the operators using golang SDK and high capability level on OperatorHub mark as stable.